export/export_pipeline/compute_collision_representation.py

```diff
@@ -0,0 +1,427 @@
+                    ff = frozenset(f)  # frozenset() also sorts
+                    if ff in fs_set:
+                        fs_set.remove(ff)
+                    else:
+                        fs_set.add(ff)
+                        final_fs.append(f)
+                mesh = trimesh.Trimesh(vs, final_fs)
+                if mesh.is_volume:
+                    mesh_parts.append(mesh)
+
+            object_assembled = trimesh.Scene()
+            np.random.seed(0)
+            i = 0
+            for p in mesh_parts:
+                p.visual.vertex_colors[:, :3] = (np.random.rand(3) * 255).astype(np.uint8)
+                object_part = trimesh.Scene()
+                object_part.add_geometry(p)
+                filename = osp.join(decompose_dir, f"{i:03d}{params['common']['CONVEX_DECOMPOSITION_EXT']}")
+                object_part.export(filename)
+                object_assembled.add_geometry(p)
+                i+=1
+            if not object_assembled.is_empty:
+                object_assembled.export(osp.join(decompose_dir, 'assembled.obj'))
+                decomposed = True
+        elif acd_args['decompose_method'] == 'vhacd':
+            if not vhacd_args['enable']:
+                return False
+
+            if shutil.which('TestVHACD') is None:
+                print(
+                    "V-HACD was enabled but not found in the system path. Either install it "
+                    "manually or run `bash install_vhacd.sh`. Skipping decomposition"
+                )
+                return False
+
+            # Call V-HACD, suppressing output.
+            fill_mode = {0: 'floor', 1: 'surface', 2: 'raycast'}
+            ret = subprocess.run(
+                [
+                    shutil.which('TestVHACD'),
+                    osp.join('convex_decomposition', mesh_filename),
+                    "-i",
+                    "stl",
+                    "-o",
+                    params['common']['CONVEX_DECOMPOSITION_EXT'][1:],
+                    "-h",
+                    f"{vhacd_args['max_output_convex_hulls']}",
+                    "-r",
+                    f"{vhacd_args['voxel_resolution']}",
+                    "-e",
+                    f"{vhacd_args['volume_error_percent']}",
+                    "-d",
+                    f"{vhacd_args['max_recursion_depth']}",
+                    "-s",
+                    f"{int(not vhacd_args['disable_shrink_wrap'])}",
+                    "-f",
+                    f"{vhacd_args['fill_mode'].name.lower()}",
+                    "-v",
+                    f"{vhacd_args['max_hull_vert_count']}",
+                    "-a",
+                    f"{int(not vhacd_args['disable_asyn'])}",
+                    "-l",
+                    f"{vhacd_args['min_edge_length']}",
+                    "-p",
+                    f"{int(vhacd_args['split_hull'])}",
+                ],
+                check=True,
+                cwd=mesh_dir,
+                stdout=subprocess.PIPE,
```

Comment:
I believe `PIPE` and `STDOUT` are the defaults and therefore do not need to be specified.